packages/starknet-snap/src/utils/snapUtils.ts

```diff
@@ -220,6 +220,57 @@ export function getSigningTxnText(
   return tokenTransferComponents1.concat(tokenTransferComponents2);
 }
 
+export function getDeclareSnapTxt(
+  senderAddress: string,
+  network: Network,
+  contractPayload: DeclareContractPayload,
+  invocationsDetails?: InvocationsDetails,
+) {
+  const components = [];
+  components.push(text('**Network:**'));
```

Comment:
Maybe add a function to avoid repeating the code : 
```
function addComponent(components, label, value) {
  components.push(text(label));
  components.push(copyable(value));
}
addComponent(components, '**Network:**', network.name);

Reply:
will be update for all after those new method PR merge

Reply:
LGTM, just need to fix conflict :)